Spanner/tests/System/OperationsTest.php

```diff
@@ -26,31 +26,31 @@
  */
 class OperationsTest extends SpannerTestCase
 {
+    use DatabaseRoleTrait;
+
     private static $id1;
     private static $id2;
-    private static $id3;
-    private static $id4;
-    private static $id5;
     private static $name1;
     private static $name2;
+    private static $isSetup = false;
 
     public static function setUpBeforeClass(): void
     {
-        self::$id1 = rand(1000, 9999);
-        self::$id2 = rand(1, 999);
-        self::$id3 = rand(10000, 10999);
-        self::$id4 = rand(11000, 11999);
-        self::$id5 = rand(12000, 12999);
-        self::$name1 = uniqid(self::TESTING_PREFIX);
-        self::$name2 = uniqid(self::TESTING_PREFIX);
-
-        parent::setUpBeforeClass();
-
-        self::$database->insert(self::TEST_TABLE_NAME, [
-            'id' => self::$id1,
-            'name' => self::$name1,
-            'birthday' => new Date(new \DateTime('2000-01-01'))
-        ]);
+        if (!self::$isSetup) {
```

Comment:
same here.. return immediately if the setup is true. 
Else set it up outside the `if` block.

Reply:
Done